dev/openchpl_views.sql

```diff
@@ -575,6 +577,7 @@ SELECT cp.certified_product_id,
        curesupdate.cures_update,
        decert.decertification_date,
        certs_with_api_documentation.cert_number AS api_documentation,
+       certs_with_service_base_url_list.cert_number AS service_base_url_list,
```

Comment:
Similar comment as on the API - if this isn't getting displayed in any collections page I'm not sure that it's needed here

Reply:
In conversation with the PO, this new column needed to be added to the "API Information..." collection. I updated the ticket requirements